packages/calypso-e2e/src/lib/pages/gutenberg-editor-page.ts

```diff
@@ -33,23 +32,39 @@ const selectors = {
 
 /**
  * Represents an instance of the WPCOM's Gutenberg editor page.
- *
- * @augments {BaseContainer}
  */
-export class GutenbergEditorPage extends BaseContainer {
-	frame!: Frame;
+export class GutenbergEditorPage {
+	private page: Page;
+
 	/**
-	 * Overrides the function of same name defined in the base class.
-	 * This ensures the iframe containing the editor is is fully loaded prior to
-	 * continuing with the test.
+	 * Constructs an instance of the component.
 	 *
-	 * @returns {Promise<void>} No return value.
+	 * @param {Page} page The underlying page.
 	 */
-	async _postInit(): Promise< void > {
-		const elementHandle = await this.page.waitForSelector( selectors.editorFrame );
-		this.frame = ( await elementHandle.contentFrame() ) as Frame;
+	constructor( page: Page ) {
+		this.page = page;
+	}
+
+	/**
+	 * Initialization steps to ensure the page is fully loaded.
+	 *
+	 * @returns {Promise<Frame>} iframe holding the editor.
+	 */
+	async waitUntilLoaded(): Promise< Frame > {
+		const frame = await this.getEditorFrame();
 		await this.page.waitForLoadState( 'networkidle' );
```

Comment:
Should we put this call above `getEditorFrame()`? Is it worth letting things fully settle before trying to do any kind of selecting?

Reply:
Ah yes, good find!